docs/site/migration/models/operation-hooks.md

```diff
@@ -6,8 +6,38 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/migration-models-operation-hooks.html
 ---
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3952](https://github.com/strongloop/loopback-next/issues/3952)
-" %}
+Operation hooks are not supported in LoopBack 4 yet. It will be, eventually; you
```

Comment:
Are we really going to have operation hooks in LB4? Maybe reword it as:
`See the [Operation hooks for models/repositories spike](https://github.com/strongloop/loopback-next/issues/1919) for the progress`

Reply:
Updated as per suggestion.